dev_guide/console.adoc

```diff
@@ -69,34 +69,3 @@ image:console_filter_step3.png["Console Filter Step 3"]
 
 image:console_filter_active.png["Console Filter Active"]
 ====
-
-
-== Creating from Template
```

Comment:
@bparees Sorry, not too sure what you mean here too. Alex mentioned above that Bilhar's suggested that we split up the console doc and put all the bits into other topics, so the Creating from a Template section will be in this section after (if) this PR is merged. 

Do you mean we should be linking to other parts of the docs to show what else can be created from templates?